webservice/template/admin/faq/index.tmpl

```diff
@@ -5,7 +5,7 @@
     <h2 class="display-4 mb-3" style="float:left;">Frequently Asked Questions <a href="/admin/faq/edit"
                                                                                  class="btn btn-success btn-sm px-3">Edit</a>
     </h2>
-    <p class="text-right" style="margin-top: 20px"><strong>Last Updated:</strong> {{ PRETTYTIME .Updated.Date }}</p>
+    <p class="text-right" style="margin-top: 20px"><strong>Last Updated:</strong> {{ PRETTYTIME .Updated.Date }} <i>(Norwegian time!)</i></p> <!-- TODO time-norwegian -->
```

Comment:
Remove "(Norwegian time!)"

Reply:
Christopher asked for this specific, so nah